pkg/model/config/config_test.go

```diff
@@ -31,6 +31,43 @@ var _ = Describe("PluginConfig", func() {
 		Data2 string `json:"data-2"`
 	}
 
+	resource := Data{Group: "Foo", Kind: "Baz", Version: "v1"}
+	resource.Webhooks = Webhooks{false, false, true}
+
+	It("should return a new Webhooks", func() {
+		Expect(resource.Group).To(BeIdenticalTo("Foo"))
+		Expect(resource.Kind).To(BeIdenticalTo("Baz"))
+		Expect(resource.Version).To(BeIdenticalTo("v1"))
+		Expect(resource.Webhooks.Defaulting).To(BeIdenticalTo(false))
+		Expect(resource.Webhooks.Validation).To(BeIdenticalTo(false))
+		Expect(resource.Webhooks.Conversion).To(BeIdenticalTo(true))
+	})
```

Comment:
This is not testing anything, you are not calling any function or methid, you just assign and check that they are assigned.

Reply:
Good cather. After the changes made it is no longer required.